src/utils.js

```diff
@@ -330,7 +330,7 @@ export function ensurePluginOrder(plugins, befores, plugin, afters) {
     const afterIndex = plugins.findIndex(plugin => plugin.pluginName === after)
     if (afterIndex < pluginIndex) {
```

Comment:
It seems check for `-1` is also needed here, whats ur take on this @tannerlinsley ?

Reply:
Yeah. It is. I'll merge this PR, then make a few more changes, including getting tests to pass now that this is working better :)